types/signed_header_test.go

```diff
@@ -129,7 +129,7 @@ func testValidateBasic(t *testing.T, untrustedAdj *SignedHeader, privKey ed25519
 		{
 			prepare: func() (*SignedHeader, bool) {
 				untrusted := *untrustedAdj
-				untrusted.Version.App = untrusted.Version.App + 1
+				untrusted.BaseHeader.Height = untrusted.BaseHeader.Height + 1
```

Comment:
This change is probably unnecessary? This is just a mutation to ensure sig verification fails?

Reply:
we need to change it because validator only signed in CometBFTVote not the entire header hash. CometBFT Vote doesn't contain Version.App.

Reply:
Makes sense! Feel free to resolve this comment :)